scikeras/wrappers.py

```diff
@@ -595,6 +597,8 @@ def _fit(self, X, y, sample_weight=None, warm_start=False):
         Raises:
             ValueError : In case of invalid shape for `y` argument.
         """
+        # Handle epochs
+        self._epochs = epochs or self.epochs
```

Comment:
Well it's either got to be an attribute, or we need to put it into the signature of `_fit` _and_ `_fit_keras_model` and propagate it all the way to `model_.fit`. I'm open to either.

But we may also have to add `classes` from #104, so I'm afraid that if we go the parameter propagation route we're going to end up with complicated function signatures. If this were external I'd preferrer that (to avoid side effects) but since this is an internal implementation, I thought attributes would be better.

Happy to change it if you disagree.

Reply:
Changed to a required parameter of the functions, no longer an attribute.

Reply:
I much prefer passing parameters with function calls then setting them as attributes. Function parameters have a clear data flow: it's possible to know exactly where they come from. Attributes do not have that feature.